### PR TITLE
WIP: Sync TEMPLATE made with nf-core tools 2.14.1

### PR DESCRIPTION
**Pull request by maxulysse**

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
+# nf/aggregate: Contributing Guidelines
```
Comment (maxulysse) [inline .github/CONTRIBUTING.md:1]:
```suggestion
# seqeralabs/nf-aggregate: Contributing Guidelines
```

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+Hi there!
+Many thanks for taking an interest in improving nf/aggregate.
```
Comment (maxulysse) [inline .github/CONTRIBUTING.md:4]:
```suggestion
Many thanks for taking an interest in improving seqeralabs/nf-aggregate.
```

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+We try to manage the required tasks for nf/aggregate using GitHub issues, you probably came to this page when creating one.
```
Comment (maxulysse) [inline .github/CONTRIBUTING.md:6]:
```suggestion
We try to manage the required tasks for seqeralabs/nf-aggregate using GitHub issues, you probably came to this page when creating one.
```

File: .github/CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+Please use the pre-filled template to save time.
+
+However, don't be put off by this template - other more general issues and suggestions are welcome!
+Contributions to the code are even more welcome ;)
+
+## Contribution workflow
+
+If you'd like to write some code for nf/aggregate, the standard workflow is as follows:
```
Comment (maxulysse) [inline .github/CONTRIBUTING.md:14]:
```suggestion
If you'd like to write some code for seqeralabs/nf-aggregate, the standard workflow is as follows:
```

File: README.md [hunk 5]
```diff
@@ -1,6 +1,16 @@
...
-Nextflow pipeline to aggregate pertinent metrics across pipeline runs on the Seqera Platform.
+[![Nextflow](https://img.shields.io/badge/nextflow%20DSL2-%E2%89%A523.10.0-23aa62.svg)](https://www.nextflow.io/)
+[![run with conda](http://img.shields.io/badge/run%20with-conda-3EB049?labelColor=000000&logo=anaconda)](https://docs.conda.io/en/latest/)
+[![run with docker](https://img.shields.io/badge/run%20with-docker-0db7ed?labelColor=000000&logo=docker)](https://www.docker.com/)
+[![run with singularity](https://img.shields.io/badge/run%20with-singularity-1d355c.svg?labelColor=000000)](https://sylabs.io/docs/)
+[![Launch on Seqera Platform](https://img.shields.io/badge/Launch%20%F0%9F%9A%80-Seqera%20Platform-%234256e7)](https://cloud.seqera.io/launch?pipeline=https://github.com/seqeralabs/nf-aggregate)
+
+## Introduction
+
+**seqeralabs/nf-aggregate** is a nextflow pipeline to aggregate pertinent metrics across pipeline runs on the Seqera Platform.
```
Comment (maxulysse) [inline README.md:13]:
```suggestion
**seqeralabs/nf-aggregate** is a Nextflow pipeline to aggregate pertinent metrics across pipeline runs on the Seqera Platform.
```

File: CITATIONS.md [hunk 6]
```diff
@@ -10,6 +10,10 @@
 
 ## Pipeline tools
 
+- [MultiQC](https://pubmed.ncbi.nlm.nih.gov/27312411/)
+
+  > Ewels P, Magnusson M, Lundin S, Käller M. MultiQC: summarize analysis results for multiple tools and samples in a single report. Bioinformatics. 2016 Oct 1;32(19):3047-8. doi: 10.1093/bioinformatics/btw354. Epub 2016 Jun 16. PubMed PMID: 27312411; PubMed Central PMCID: PMC5039924.
+
```
Comment (drpatelh) [inline CITATIONS.md:16]:
```suggestion

- [Seqera CLI](https://github.com/seqeralabs/tower-cli)

```

**Comment by maxulysse** [comment]
closed in favor of #39

**Pull request closed without merging**